src/cyrillux-usemarcon-replacement.js

```diff
@@ -0,0 +1,409 @@
+    replace(/fold\.? maps/gu, 'folded maps').
+    // replace(/\bmin\./gu, 'minutes').
+    // replace(/\bmin\b/gu, 'minutes').
+    replace(/\bp\.\b/gu, 'pages').replace(/\bp\b/gu, 'pages').
+    replace(/\bpbk\.\b/gui, 'paperback').replace(/\bpbk\b/gui, 'paperback'). // expand to MTS-compliant form
+    replace(/\bports\./gu, 'portraits').
+    replace('sd., col.', 'sound, color').
+    replace(/ *\((?:chiefly col\.|chiefly color|some cols)\)/gu, '').
+    replace(/\b1 hr\./gu, '1 hour');
+}
+
+function expandFinnishAbbreviations(value) {
+  return value.replace(/\bcn\. /gu, 'noin ').
+    // replace(/\bmin\./gu, 'minuuttia').
+    // replace(/\bmin\b/gu, 'minuuttia').
+    replace(/\bnid\./gu, 'nidottu').replace(/\bnid\b/gu, 'nidottu').
+    replace(/\bsid\./gu, 'sidottu').replace(/\bsid\b/gu, 'sidottu').
+    replace(/^\(([^)]+)\)$/u, '$1'); // eslint-disable-line prefer-named-capture-group
+  // <- removal of brackets above could use a better location
+}
+
+function expandSwedishAbbreviations(value) {
+  return value.replace(/\bca\. /gu, 'circa ').
+    replace(/\bhft\./gui, 'häftad').replace(/\bhft\b/gui, 'häftad');
+  // replace(/\bmin\./gu, 'minuter').
+  // replace(/\bmin\b/gu, 'minuter');
+}
+
+function finnishTranslationsAndMappings(value) {
+  return value.replace('analog', 'analoginen').
+    replace('approximately', 'noin').
+    replace('audio discs', 'äänilevyä').
+    replace('black and white', 'mustavalkoinen').
+    replace(/\bcharts\b/gu, 'kaavioita').
+    replace('chiefly color illustrations', 'pääosin värikuvitettu').
+    replace('chiefly', 'pääosin').
+    replace('color illustrations', 'värikuvitus').
+    replace(/\bdigital\b/gu, 'digitaalinen').
+    replace(/\belectronic book\b/gu, 'verkkoaineisto').
+    replace(/\bfolded sheet\b/gu, 'taitelehti').
+    replace(/\bhardback\b/gu, 'kovakantinen').
+    replace(/\bhours\b/gu, 'tuntia').
+    replace('illustrated', 'kuvitettu').
+    replace(/illustratation\b/gu, 'kuvitus'). // kuvitus/kuvitettu variation is taken from usemarcon-cyrillux...
+    replace('illustratations', 'kuvitettu').
+    replace(/\binsert\b/gu, 'liite').
+    replace(/\binserts\b/gu, 'liitteitä').
+    replace('leaves of plates', 'kuvalehteä').
+    replace(/\bmap\b/gu, 'kartta').
+    replace(/\bmaps\b/gu, 'karttoja'). // or karttaa?
+    replace('minutes', 'minuuttia').
+    replace('online resource', 'verkkoaineisto').
+    replace('pages of plates', 'kuvalehteä').
+    replace(/\bpages\b/gu, 'sivua').
+    replace(/\bpaperback\b/gu, 'pehmeäkantinen'). // MTS alt
+    replace(/\bsoftcover\b/gu, 'pehmeäkantinen'). // MTS pref
+    replace('sound, color', 'äänellinen, värillinen').
+    replace('sound cassettes', 'äänikasettia').replace('sound cassette', 'äänikasetti').
+    replace('sound discs', 'äänilevyä').replace('sound disc', 'äänilevy').
+    replace('unnumbered', 'numeroimatonta').
+    replace(/\bverkojulk\.\b/gu, 'verkkoaineisto').replace(/\bverkojulk\b/gu, 'verkkoaineisto').
```

Comment:
Are we missing a 'k' from verk**k**ojulk here - or fixing a systemic typo

Reply:
done